### PR TITLE
Add aarch64-pc-windows-msvc building target for nu binaries

### PR DESCRIPTION
**Pull request by hustcer**
# Description

1. Add `aarch64-pc-windows-msvc` building target for nu binaries
2. Bump nushell version for the binary release script

# User-Facing Changes

Add two more packages:

1. aarch64-pc-windows-msvc.zip
2. aarch64-pc-windows-msvc.msi

# Tests + Formatting

Release workflow test: https://github.com/hustcer/nu-release/actions/runs/4943037199
Create release draft test: https://github.com/hustcer/nu-release/releases/tag/v0.79.2

### CODE REVIEW

File: .github/workflows/release-pkg.nu [hunk 1]
```diff
@@ -99,6 +99,7 @@ if $os in [$USE_UBUNTU, 'macos-latest'] {
 # Build for Windows without static-link-openssl feature
 # ----------------------------------------------------------------------------
 if $os in ['windows-latest'] {
+    # let-env CARGO_BUILD_TARGET = $target
```
Comment (rgwood) [inline .github/workflows/release-pkg.nu:102]: Is this commented-out line intentional?

File: .github/workflows/release-pkg.nu [hunk 2]
```diff
@@ -128,7 +129,7 @@ sleep 3sec
 
 print $'(char nl)Check binary release version detail:'; hr-line
 let ver = if $os == 'windows-latest' {
-    (do -i { ./output/nu.exe -c 'version' }) | str join
+    (do -i { .\output\nu.exe -c 'version' }) | str join
```
Comment (rgwood) [inline .github/workflows/release-pkg.nu:132]: Was this not working before?

File: .github/workflows/release-pkg.nu (hunk 1 above)
Comment (hustcer) [reply .github/workflows/release-pkg.nu:102]: Yes, it is.

File: .github/workflows/release-pkg.nu (hunk 2 above)
Comment (hustcer) [reply .github/workflows/release-pkg.nu:132]: It works before, but it's the right way to do it on Windows, so it's kept.

File: .github/workflows/release-pkg.nu (hunk 1 above)
Comment (rgwood) [reply .github/workflows/release-pkg.nu:102]: What is it for?

Comment (hustcer) [reply .github/workflows/release-pkg.nu:102]: The original script works too good to be true, I'm just trying another way to specify the cargo build target to make sure it works. Of course, you can remove it.

**Review by rgwood — APPROVED** [review]

**Comment by fdncred** [comment]
This PR broke the winget release because we now have 2 msi files and therefore we need 2 winget releases. I guess you can only have 1 msi per winget_pkg? See https://github.com/nushell/nushell/pull/9322 and https://github.com/nushell/nushell/pull/8070

**Comment by hustcer** [comment]
I see, I will try to fix it before the next release

**Comment by fdncred** [comment]
@hustcer Thanks. One of the people who helped create the winget release thinks it may be fine, it's just that we've never released 2 msi files before. You should read the threads i posted to see if something needs to be done or not. Here's the current winget release for reference on what it should look like https://github.com/microsoft/winget-pkgs/pull/109445

**Comment by hustcer** [comment]
> Once the PR is merged, the action should now work with the next releases. It checks if the installer URL count match with the previous release.

Yeah, I agree with @sitiom https://github.com/nushell/nushell/actions/runs/5226501091/jobs/9437222070 failed because it ran `java -jar komac.jar update` with two msi urls and we don't have aach64 msi before. It should work after https://github.com/microsoft/winget-pkgs/pull/109445 been merged

**Pull request merged**